content/docs/projects/csi-driver-spiffe.md

```diff
@@ -0,0 +1,234 @@
+---
+title: csi-driver-spiffe
+description: ''
+---
```

Comment:
note: Clearly copied directly from the csi-driver-spiffe README so I'm not going to dig into it too much :+1: